Actors/ActorCounter/Scripts/uninstall-sfctl.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sfctl application delete --application-id CounterActorApplication
+sfctl application unprovision --application-type-name CounterActorApplicationType --application-type-version 1.0.0
+sfctl store delete --content-path CounterActorApplication
```

Comment:
I was just trying out the commands.. seems this 'store delete' thing is not working. It gave the following error - 
az sf: error: argument subcommand: invalid choice: store